SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Supported Versions
+
+<!-- Use this section to tell people about which versions of your project are
+currently being supported with security updates. -->
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.6.x   | :white_check_mark: |
+| < 1.0.0 | :x:                |
+
+## Reporting a Vulnerability
+
+<!-- Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc. -->
+
+To report a security issue, please email
+[email@example.com](mailto:email@example.com)
+with a description of the issue, the steps you took to create the issue,
+affected versions, and, if known, mitigations for the issue.
+
+We will respond and solve it as soon as possible.
```

Comment:
Should it be something like: "All support will be made on the best effort base, so please indicate the "urgency level" of the vulnerability as Critical, High, Medium or Low."

This could help you on tracking which deserves most effort and which does not

Reply:
Sounds good to me.